frontend/views/containers/user-settings/FontSize.vue

```diff
@@ -39,26 +39,24 @@ export default {
     ])
   },
 
+  watch: {
+    fontSize () {
+      this.$refs.slider.refresh()
+    }
+  },
+
   created () {
-    let size = 57.5
+    let size = 12
     for (let index = 0; index < this.fontVariations; index++) {
       // Add data to the slider
       this.fontData.push(size)
-      size += 2.5
+      size += 2
 
       // Hise one label out of two
       this.fontRange.push({
-        label: 'Aa',
-        isHide: index % 2
+        label: 'Aa'
```

Comment:
It's odd that when clicking in a white space somewhere in the input range, that triggers the font-size change. IMO it should only change the font-size when we click exactly at "aA" or at the input range bar.